### PR TITLE
VBWKDE resolutions, superficial changes to make code more PEP 8

### PR DESCRIPTION
**Pull request by jllanfranchi**

### CODE REVIEW

**Comment by jllanfranchi** [comment]
Also note the Reco.py smearing step has the 5-nested-for-loops replaced by 2 nested loops and a call to np.tensordot(). Note that this could impact behavior, but I tested via template_checks.py before and after the change and didn't notice any difference in the outputs. I forgot to mention that in the commit message.

**Comment by sboeser** [comment]
Hi Tim,
I've gotta few small changes coming up to this one to fix some installation issues. So please hold off before you merge...

**Comment by tarlen5** [comment]
No problem, Sebastian. I'll wait til your updates.

**Comment by jllanfranchi** [comment]
Sebastian, see if you can get the code specifically from my commit 5544c3a. I made a few changes besides just fixing the setup.py/MANIFEST.in and removing the hard-coded path in the hash_file function (I'm using a different field in the JSON template settings file, e.g.). The only way I could figure out, with my limited knowledge, to include my gaussians.c file in the setup.py was to comment out the line
#ext_package='pisa.oscillations.prob3',

Feel free to modify from there, I started learning about Python installation package management 30 minutes before I submitted that change... :)

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1,3 @@
 include pisa/oscillations/prob3/*.h
+include pisa/utils/gaussians.pyx
+include pisa/utils/gaussians.c
```
Comment (tarlen5) [inline MANIFEST.in]: @sboeser @jllanfranchi  In this PR, gaussians.c is not included, even though it is shown here. Based on the discussion on the call today, what we (Justin) will do is remove the compilation stage of the gaussians module and we'll simply make cython an optional package to install and use for this stage (and for future pisa development if desired).

**Pull request merged**